script/translate.go

```diff
@@ -102,12 +106,16 @@ func inTranslate(n *html.Node, filename string) {
 	}
 }
 
-func translation(v string) {
+func translation(v string, k string) {
```

Comment:
I don't think we need a separate key here.  The `v` argument holds the key, which also serves as the initial translation unless there is a better one.